models/category.model.js

```diff
@@ -5,4 +5,18 @@ exports.selectCategories = () => {
         .then((categories) => {
             return categories.rows
         })
+}
+
+exports.selectReviewById = (id) => {
+    return DB.query(`SELECT * from reviews WHERE review_id = $1`, [id]).then((review) => {
```

Comment:
this is very nit-picky but `db.query` returns a promise that resolves with a `Result` object - naming the arrow functions parameter `reviews` (or `categories` from above) is a bit misleading

Reply:
cool, should I resolve all these things on this branch or do I have to restore previous ones?